src/_gettsim/transfers/rente.py

```diff
@@ -463,37 +444,33 @@ def _ges_rente_altersgrenze_abschlagsfrei_mit_besond_langj(  # noqa: PLR0913
     ----------
     ges_rente_regelaltersgrenze
         See :func:`ges_rente_regelaltersgrenze`.
-    ges_rente_frauen_altersgrenze
-        See :func:`ges_rente_frauen_altersgrenze`.
+    _ges_rente_frauen_altersgrenze
+        See :func:`_ges_rente_frauen_altersgrenze`.
     _ges_rente_langj_altersgrenze
         See :func:`_ges_rente_langj_altersgrenze`.
     _ges_rente_besond_langj_altersgrenze
         See :func:`_ges_rente_besond_langj_altersgrenze`.
     _ges_rente_arbeitsl_altersgrenze
         See :func:`_ges_rente_arbeitsl_altersgrenze`.
-    ges_rente_vorauss_regelrente
-        See :func:`ges_rente_vorauss_regelrente`.
     ges_rente_vorauss_frauen
         See :func:`ges_rente_vorauss_frauen`.
     ges_rente_vorauss_langj
         See :func:`ges_rente_vorauss_langj`.
     ges_rente_vorauss_besond_langj
         See :func:`ges_rente_vorauss_besond_langj`.
-    _ges_rente_vorauss_arbeitsl
-        See :func:`_ges_rente_vorauss_arbeitsl`.
+    ges_rente_vorauss_arbeitsl
+        See :func:`ges_rente_vorauss_arbeitsl`.
 
     Returns
     -------
     Full retirement age.
 
     """
 
-    out = float("inf")
```

Comment:
The default value of `inf` doesn't make much sense to me (but maybe I'm missing something here?). The default retirement age without deductions should be the Regelaltersgrenze (NRA).

Reply:
But this here already checks whether people qualify, right? Tbh, I am not sure what the purpose is in the end -- I would think we'd check this as part of the `voraussetzungen`? Or am I misinterpreting the abbreviation?

Reply:
The function is supposed to yield the earliest age at which the person can claim benefits without any deductions. The `float("inf")` is just the value that is returned if the person is not eligible for any pathway into retirement. 

I'm arguing, that the function should return the NRA in that case, because it feels a bit stupid to set all statutory retirement ages to `inf` if the person is not eligible for retirement (in that case, we do the eligibility check downstream). I stumbled on this when I used these targets on real data, seems very unnatural to me.

Reply:
One can argue for most stuff. From my point of view, "eligible with current state variables" is the  only important thing, we can't project into the future much, anyhow. So I am not fully sure why we need this at all...

Put differently, I think the cleanest thing is inf in case we are talking about a future age, but almost anything that is consistent and well documented is fine with me.

Reply:
Put differently, I don't know what "eligible **at all**" would encompass. Anything but age? Some other selection?

Reply:
We are talking about current age.

This functions and all the others where I changed the default outputs (if all eligibility checks fail) are inputs of the Zugangsfaktor calculation. To make the changes that I favour explicit this would be the logic of the Zugangsfaktor calculation:

1. Calculate for each pathway into retirement the statutory retirement age (without any eligibility checks, but conditional on birth year).
2. Compute the earliest possible retirement age `_ges_rente_altersgrenze_vorzeitig`. This checks for eligibility of the pathways that allow for early retirement, but not for elgibility for Regelaltersrente. Hence, if the individual is not eligible for early retirement, it returns the NRA. **I didn't change anything here**.
3. Compute the reference age for deductions `_ges_rente_altersgrenze_abschlagsfrei`. Here I want to use the same logic as in 2: Compute the reference age, checking for eligibility for all pathways that allow for early retirement, but use the NRA if the individual is not eligible for any of those pathways. No eligibility check for Regelaltersrente.
4. Compute the Zugangsfaktor. Return 0 if not `rentner` or if not eligible for Regelaltersrente.

Reply:
Cool. Would just be good if that logic became very apparent somewhere!